posts/models.py

```diff
@@ -20,3 +39,21 @@ class Meta:
         verbose_name = 'sendings'
         verbose_name_plural = 'sendings'
 
+
+class Like(models.Model):
+    user = models.ForeignKey(User, on_delete=models.CASCADE)
+    post = models.ForeignKey(Post, on_delete=models.CASCADE)
+
+    date_created = models.DateTimeField(auto_now_add=True)
+
+
+class Rate(models.Model):
+    user = models.ForeignKey(User, on_delete=models.CASCADE)
+    post = models.ForeignKey(Post, on_delete=models.CASCADE)
+
+    RATINGS = ((1, '1'), (2, '2'), (3, '3'), (4, '4'), (5, '5'))
+    rating = models.IntegerField(max_length=1, choices=RATINGS, default=5)
```

Comment:
We can use `score` instead of `rating`. What do you think ? `rating` in `Rate` model is not good.